### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/tutorial/first_steps.md`

### PR DESCRIPTION
**Pull request by Kadermiyanyedi**
I translated tutorial/first_steps.md to Turkish.

Main issue: https://github.com/tiangolo/fastapi/issues/1892

### CODE REVIEW

**Review by BilalAlpaslan — CHANGES_REQUESTED** [review]

File: docs/tr/docs/tutorial/first_steps.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+#### Data "şema"
+
+"Şema" terimi, JSON içeriği gibi bazı verilerin şeklini de ifade edebilir.
+
+Bu durumda, JSON öznitelikleri ve sahip oldukları veri türleri vb. anlamına gelir.
+
+#### OpenAPI and JSON Şema
+
+OpenAPI, API'niz için bir API şeması tanımlar. Ve bu şema, JSON veri şemaları standardı olan **JSON Şema** kullanılarak API'niz tarafından gönderilen ve alınan verilerin tanımlarını (veya "şemalarını") içerir.
```
[the remark this answers is not in the document]
Comment (Kadermiyanyedi) [reply docs/tr/docs/tutorial/first_steps.md:90]: Buna gerek yok bence^^

**Comment by Kadermiyanyedi** [comment]
@BilalAlpaslan  güncellemeleri gönderdim. Tekrar review edebilirsiniz :)

**Review by BilalAlpaslan — APPROVED** [review]

**Review by onuralpszr — APPROVED** [review]
Lgtm again (i did my review originally in previous pr)

**Comment by onuralpszr** [comment]
friendly ping  @tiangolo maybe a merge time  :)

**Comment by tiangolo** [comment]
Awesome, thank you @Kadermiyanyedi! :cake: :clap: 

And thanks for the reviews @onuralpszr and @BilalAlpaslan :bow:

**Pull request merged**